src/lib/blockchain.js

```diff
@@ -1,8 +1,9 @@
 import Web3 from 'web3';
 import contractABIs from './contracts';
 import eventContracts from './eventContracts';
+import config from '../config';
 
-const port = 8545;
+const port = config('port');
```

Comment:
This would be:

`const port = config('default_port');`

So in the future when we can config this from a command line argument we can have:

`const port = args['port'] || config('default_port');`